### PR TITLE
PDR-3726 kafka reader

### PR DESCRIPTION
**Pull request by wangtuo**
## Changes
   
  - [ ] kafka reader
  - [ ] 已经做了手工集成测试
  
 
## Reviewers

  - [x] @wonderflow  please review
  - [ ] @lvheyang please review
  - [ ] @visaxin please review
    
## Checklist
   
   - [ ] Rebased/mergable
   - [ ] Tests pass
   - [ ] CHANGELOG.md updated
   - [ ] Jira issue/task done

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -368,6 +369,28 @@ reader 的典型配置如下
...
+
+Kafka reader 的典型配置如下
+
+```
+    "reader":{
+        "mode": "kafka",
+        "kafka_groupid":"mac1",
+        "kafka_topic":"test_topic1，test_topic2",
+        "kafka_zookeeper":"localhost:2181, localhost:2181, localhost:2181",
+        "read_from":"oldest"
```
Comment (wonderflow) [inline README.md]: read_from下面也提一下吧，其他的不支持的

File: reader/kafka.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+		Whence:         whence,
+		readChan:       make(chan json.RawMessage),
+		errs:           make(chan error, 1000),
+		status:         StatusInit,
+		mux:            sync.Mutex{},
+		started:        false,
+	}
+	return kr, nil
+}
+func (kr *KafkaReader) Name() string {
```
Comment (wonderflow) [inline reader/kafka.go]: 这个也增加一个测试吧，提高覆盖率

File: README.md (hunk 1 above)
Comment (wangtuo) [inline README.md]: done

File: reader/kafka.go (hunk 2 above)
Comment (wangtuo) [inline reader/kafka.go]: 有测试的

**Comment by wangtuo** [comment]
@wonderflow fix done

**Pull request merged**